exp-player/addon/components/exp-rating-form/component.js

```diff
@@ -264,19 +264,19 @@ var questions = [
             formatLabel: 'format-label',
             labels: [
                 {
-                    rating: 0,
+                    rating: 'number1',
```

Comment:
I think the numbers should get serialized as the actual number (e.g. 1 instead of "number1").